libflux/flux/src/lib.rs

```diff
@@ -1119,21 +1119,30 @@ from(bucket: v.bucket)
-        let got = format!("{:?}", d[20]); // the csv DocPackage location
+        let doc = docs();
+        let stat = r#"PackageDoc { name: "csv", headline: "Package csv provides tools for working with data in annotated CSV format.", description: None, members: {"from": Function(FunctionDoc { name: "from", headline: "from is a function that retrieves data from a comma separated value (CSV) data source. ", description: "A stream of tables are returned, each unique series contained within its own table. Each record in the table represents a single point in the series. ## Query anotated CSV data from file\n```\nimport \"csv\"\n\ncsv.from(file: \"path/to/data-file.csv\")\n```\n\n## Query raw data from CSV file\n```\nimport \"csv\"\n\ncsv.from(\n  file: \"/path/to/data-file.csv\",\n  mode: \"raw\"\n)\n```\n\n## Query an annotated CSV string\n```\nimport \"csv\"\n\ncsvData = \"\n#datatype,string,long,dateTime:RFC3339,dateTime:RFC3339,dateTime:RFC3339,string,string,double\n#group,false,false,false,false,false,false,false,false\n#default,,,,,,,,\n,result,table,_start,_stop,_time,region,host,_value\n,mean,0,2018-05-08T20:50:00Z,2018-05-08T20:51:00Z,2018-05-08T20:50:00Z,east,A,15.43\n,mean,0,2018-05-08T20:50:00Z,2018-05-08T20:51:00Z,2018-05-08T20:50:20Z,east,B,59.25\n,mean,0,2018-05-08T20:50:00Z,2018-05-08T20:51:00Z,2018-05-08T20:50:40Z,east,C,52.62\n\"\n\ncsv.from(csv: csvData)\n\n```\n\n## Query a raw CSV string\n```\nimport \"csv\"\n\ncsvData = \"\n_start,_stop,_time,region,host,_value\n2018-05-08T20:50:00Z,2018-05-08T20:51:00Z,2018-05-08T20:50:00Z,east,A,15.43\n2018-05-08T20:50:00Z,2018-05-08T20:51:00Z,2018-05-08T20:50:20Z,east,B,59.25\n2018-05-08T20:50:00Z,2018-05-08T20:51:00Z,2018-05-08T20:50:40Z,east,C,52.62\n\"\n\ncsv.from(\n  csv: csvData,\n  mode: \"raw\"\n)\n```\n\n", parameters: [ParameterDoc { name: "csv", headline: " is CSV data.", description: Some("Supports anonotated CSV or raw CSV. Use mode to specify the parsing mode."), required: false }, ParameterDoc { name: "file", headline: " is the file path of the CSV file to query.", description: Some("The path can be absolute or relative. If relative, it is relative to the working directory of the  fluxd  process. The CSV file must exist in the same file system running the  fluxd  process."), required: false }, ParameterDoc { name: "mode", headline: " is the CSV parsing mode. Default is annotations.", description: Some("Available annotation modes: annotations: Use CSV notations to determine column data types. raw: Parse all columns as strings and use the first row as the header row and all subsequent rows as data."), required: false }], flux_type: "" })} }"#;
+        let mut got = String::new();
+        for d in doc {
+            if d.name == "csv" {
+                got = format!("{:?}", d);
```

Comment:
When we assert on the stringified version, what we're testing is the stringifying of the object. Rather than converting to a string, `stat` should be a `PackageDoc` instance that we test against.  That was the real issue with this test.

Reply:
I understand, I will reconstruct the tests accordingly.

Reply:
The tests for `csv` and `array` have now been updated to test the `PackageDoc` structures.